static/tour/getting-started/index.html

```diff
@@ -29,39 +29,43 @@ <h3>Example</h3>
                     class="line-height--32">Run this JavaScript example</span></button>
         </div>
 
-        <div class="hidden margin-top--2" data-tour="error" role="alert">
-            <div class="font-size--16 form-error filter-overview__error-message margin-top--1 margin-bottom--1">
-                An error occurred when trying to display results. Please try again.
+        <section id="results-section" tabindex="-1" aria-labelledby="results-heading">
+            <h4 id="results-heading">Results</h4>
```

Comment:
The `h4` - "Results" - is displayed before running the JS example. If this isn't as intended, then I think we need to apply the `hidden` class and then unhide when data is fetched successfully (applies to each page of the tour)
![image](https://user-images.githubusercontent.com/23668262/93870032-8d24a500-fcc4-11ea-8404-fcd75bf31a75.png)

Reply:
Good point. Will fix this.